lang/pt-BR/spec/v1.0.0.md

```diff
@@ -0,0 +1,233 @@
+tem um monte de código dependente. O custo que deve ser incorrido para atualizar
+pode ser significante. Tendo que aumentar a versão maior para lançar mudanças
+incompatíveis, significa que você pensará no impacto das suas mudanças, e
+avaliará a relação de custo/benefício envolvida.
+
+### Documentar toda a API pública dá muito trabalho!
+
+É sua responsabilidade como desenvolvedor profissional documentar corretamente o
+software que será usado por outros. Gerenciar a complexidade de software é uma
+parte muito importante para manter o projeto eficiente, e isto é difícil de
+fazer se ninguém sabe como usá-lo ou que métodos são seguros de chamar. A longo
+prazo, Versionamento Semântico e a insistência em uma API pública bem definida
+podem deixar tudo e todos funcionamente suavemente.
+
+### O que eu faço se, acidentalmente, liberar uma mudança incompatível com versões anteriores como uma versão menor (minor version)?
+
+Assim que você perceber que quebrou a especificação de versionamento semântico,
+conserte o problema e lance uma nova versão menor, que corrige o problema e
+restaura a compatibilidade. Lembre-se que é inaceitável modificar versões
+lançadas, mesmo sob essas circustâncias. Se for apropriado, documente a versão
+ofensiva e informe seus usuários do problema de forma que eles fiquem cientes da
+versão em questão.
+
+### O que devo fazer se eu atualizar minhas próprias dependências sem modificar a API pública?
+
+Isso seria considerado compatível, uma vez que não afeta a API pública. Software
+que depende explicitamente da mesmas dependências que seu pacote, deve ter sua
+própria especificação de dependência e o autor notificará quaisquer conflitos.
+Para determinar se a mudança é a nível de correção ou modificação de nível menor
+dependente se você atualizou suas dependências a fim de corrigir um bug ou
+introduzir nova funcionalidade. Eu normalmente esperaria código adicional para
+última instância, caso em que é obviamente um incremento no nível menor.
+
+Sobre
+-----
+
+A Especificação da Semântica de Versionamento é autoria de [Tom
+Preston-Werner](http://tom.preston-werner.com), criador do Gravatars e
+co-fundador do GitHub.
+
+A tradução deste documento para Português-Brasil foi iniciada de forma
+colaborativa pela [Wend Tecnologia](https://github.com/wendtecnologia) através
+de [Walker de Alencar Oliveira](https://github.com/walkeralencar) e teve a
+participação de:
+
+- [William G. Comnisky](https://github.com/wcomnisky)
+- [Rafael Sirotheau](https://github.com/rafasirotheau)
+- [Arthur Almeida](https://github.com/arthuralmeidap)
+- [Alberto Guimarães Viana](https://github.com/albertogviana)
+- [Rafael Lúcio](https://github.com/poste9)
+- Josiel Rocha
+- Alessandro Leite
+- Vinícius Assef
+- [Silas Ribas Martins](https://github.com/silasrm)
+- [Rogerio Prado de Jesus](https://github.com/rogeriopradoj)
+- [João Lucas Farias](https://github.com/jlucfarias)
+
```

Comment:
I don't see any reason to have translation specifications/translators names here, it already exists in https://semver.org/lang/pt-BR/#sobre , would make it confusing when adding new names

Reply:
Applies to other files too